src/Compatibility/Microsoft.DotNet.ApiCompatibility/ValidationSuppressions/SuppressionEngine.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Xml.Serialization;
+
+namespace Microsoft.DotNet.ValidationSuppression
```

Comment:
Maybe we could name this, `Microsoft.DotNet.Compatibility.Suppression`?

Reply:
I can add the Compatibility as part of the namespace, but the thing I don't love about having `Suppression` as the namespace is that now we will have class `Microsoft.DotNet.Compatibility.Suppression.Suppression`, which is why I originally added ValidationSuppression as the namespace instead.

Reply:
Yeah that's not ideal. I just don't like `ValidationSuppression` cause `Validation` could apply to any form of validation and this is very specific to compatibility.

Reply:
Sure, I'll probably won't have time to finish this PR but feel free to rename as appropriate.